pkg/auto/lights/patch.go

```diff
@@ -157,6 +157,11 @@ type source struct {
 
 func (b *BrightnessAutomation) processConfig(ctx context.Context, cfg config.Root, sources []*source, changes chan<- Patcher) (sourceCount int) {
 	logger := b.logger.With(zap.String("auto", cfg.Name))
+
+	b.refreshEvery.Store(cfg.RefreshEvery.Duration)
+	b.maxRetries = cfg.OnProcessError.MaxRetries
+	b.backoffMultiplier = cfg.OnProcessError.BackOffMultiplier.Duration
```

Comment:
Consider what happens if any of these are 0 or negative.
There should be some bounds checking; there is already some validation in `config.Read` which could be extended for example.